apax/bal/api.py

```diff
@@ -82,7 +82,9 @@ def kernel_selection(
     config, params = restore_parameters(model_dir)
 
     n_train = len(train_atoms)
-    dataset = initialize_dataset(config, RawDataset(atoms_list=train_atoms + pool_atoms))
+    dataset = initialize_dataset(
+        config, RawDataset(atoms_list=train_atoms + pool_atoms), False
+    )
```

Comment:
The `config` and `RawDataset` are clear to me, but passing just `False` seems a bit odd - I'd suggest using a keyword argument instead.